controller/auth/user.go

```diff
@@ -143,6 +161,7 @@ func CreateJWT(email string, host string) ([]byte, error) {
 	claims.SetJWTID(string(encodedJwtid))
 	claims.SetNotBefore(t)
 	claims.SetSubject(subject)
+	claims.Set("eid", email)
 
 	log.Debug("JWT Claims:")
```

Comment:
It would be better (concise and self-contained) if you made an anonymous struct and used something like `log.WithFields(&Fields{<insert claims here>}).Debug("JWT Claims")`